### PR TITLE
Make SMPPs acks optional.

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
@jerith & @hodgestar ready for review. Test failing on real redis but I suspect that's due to me not waiting on some socket cleanup?

**Comment by smn** [comment]
Fixed the test with @jerith's help. Please review :)

File: vumi/transports/smpp/config.py [hunk 1]
```diff
@@ -27,6 +27,12 @@ class SmppTransportConfig(Transport.CONFIG_CLASS):
         'How long (in seconds) to wait for the SMSC to return with a '
         '`submit_sm_resp`. Defaults to 24 hours.',
         default=(60 * 60 * 24), static=True)
+    disable_ack = ConfigBool(
+        'Disable publishing of `ack` events. In some cases this event '
+        'causes more noise than signal. It can optionally be turned off. '
+        'Defaults to True.',
+        default=False, static=True
```
Comment (jerith) [inline vumi/transports/smpp/config.py]: The text and the `default` value contradict each other here. I think the text is wrong.

Also, I don't think the closing paren needs its own line. :-)

**Comment by jerith** [comment]
One comment, otherwise this looks good.

**Comment by hodgestar** [comment]
Other than @jerith's comment, looks good to me too.

**Comment by jerith** [comment]
Some of the build failures are pypy 2.5.0 and cryptography. I've restarted the failed build job that wasn't.

**Comment by hodgestar** [comment]
All builds passed except pypy.

**Comment by smn** [comment]
Is that a :+1: then?

**Comment by hodgestar** [comment]
Yep. :+1:

**Pull request merged**